pwndbg/commands/heap.py

```diff
@@ -1546,3 +1547,74 @@ def heap_config(filter_pattern: str) -> None:
             "Some config values (e.g. main_arena) will be used only when resolve-heap-via-heuristic is `auto` or `force`"
         )
     )
+
+
+# Jemalloc
+
+parser = argparse.ArgumentParser(
+    description="Returns extent information for pointer address allocated by jemalloc"
+)
+parser.add_argument("addr", type=int, help="Address of the allocated memory location")
+
+
+@pwndbg.commands.ArgparsedCommand(parser, category=CommandCategory.HEAP)
+def jemalloc_find_extent(addr) -> None:
+    print(C.banner("Jemalloc find extent"))
+    print("This command only support jemalloc 5.3.0")
```

Comment:
5.3.0 or newer too? or we dont know for sure?

Maybe: "(This command was tested only for jemalloc 5.3.0)"?

Reply:
What about "This command was tested only for jemalloc 5.3.0 and does not support lower versions"? Since we know for sure lower versions are not supported due to different structure.